Exiled.Loader/ConfigManager.cs

```diff
@@ -16,8 +16,8 @@ namespace Exiled.Loader
     using API.Enums;
     using API.Extensions;
     using API.Interfaces;
-
     using Exiled.API.Features;
+    using Exiled.API.Features.Pools;
```

Comment:
I believe this is not needed as it does not exist. Hence the build compilation error in the CI.

Reply:
https://github.com/Vladislav-CS/EXILED/pull/3